lib/src/runner/configuration/args.dart

```diff
@@ -82,6 +82,8 @@ final ArgParser _parser = (() {
   parser.addOption("timeout",
       help: 'The default test timeout. For example: 15s, 2x, none',
       defaultsTo: '30s');
+  parser.addOption('line-length',
+      help: 'Overrides the max width of lines to print.');
```

Comment:
Mark this as hidden and add a comment that it will be removed in the future and that anyone using it should notify the `test` authors so they can be warned when removal is impending.

Reply:
Done, I also added a comment with a list of current users of this feature (currently just package:build_runner) so it can be tracked.